### PR TITLE
Added MoveCard

### PR DESCRIPTION
**Pull request by johnmlee101**
@mattmolo You have the power now! MUHAHAUHAUHAHAHAH.

Just wanted to get you more involved . . . :')

Also added simple templating files for these issues. make them look nice!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -137,12 +137,38 @@ class GHProjectManager {
 	 }
 
 	/*
+	 * Moves a card over to the specified Column
+	 *
+	 * @param string owner Owner username
+	 * @param string repo Repository name
+	 * @param number issueID
```
Comment (mattmolo) [inline index.js]: the function takes in `cardID`, not `issueID`

File: index.js [hunk 2]
```diff
@@ -187,6 +213,7 @@ class GHProjectManager {
 					}
 				});
 	 		});
+	 		if (options.method == 'POST') req.write(payload);
```
Comment (mattmolo) [inline index.js]: brackets?  :P

```
    {

}
```

**Comment by johnmlee101** [comment]
@mattmolo Updated :P

**Comment by mattmolo** [comment]
LGTM

**Pull request merged**